public/app/features/explore/LogsNavigation.tsx

```diff
@@ -0,0 +1,243 @@
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [visibleRange, absoluteRange, logsSortOrder]);
+
+  useEffect(() => {
+    const index = pages.findIndex((page) => page.queryRange.to === absoluteRange.to);
+    setCurrentPageIndex(index);
+  }, [pages, absoluteRange]);
+
+  useEffect(() => {
+    const initialRange = absoluteRange.to - absoluteRange.from;
+    setRequestRange(initialRange);
+    // We want to set requestRange only when the first query is run
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  const changeTime = ({ from, to }: AbsoluteTimeRange) => {
+    setExpectedRange({ from, to });
+    onChangeTime({ from, to });
+  };
+
+  const formatTime = (time: number) => {
+    return `${dateTimeFormat(time, {
+      format: systemDateFormats.interval.second,
+      timeZone: timeZone,
+    })}`;
+  };
+
+  const oldestLogsFirst = logsSortOrder === LogsSortOrder.Ascending;
+  const theme = useTheme();
+  const styles = getStyles(theme, oldestLogsFirst, loading);
+  return (
+    <div className={styles.navContainer}>
+      {/*
+       * We are going to have 2 buttons - on the top and bottom - Oldest and Newest.
+       * Therefore I have at the moment duplicated the same code, but in the future iteration, it ill be updated
+       */}
+      {oldestLogsFirst && (
+        <Button
+          data-testid="fetchLogsTop"
+          className={styles.navButton}
+          variant="secondary"
+          onClick={() => {
+            // the range is based on initally selected range
+            changeTime({ from: visibleRange.from - requestRange, to: visibleRange.from });
+          }}
+          disabled={loading}
+        >
+          <div className={styles.navButtonContent}>
+            {loading ? <Spinner /> : <Icon name="angle-up" size="lg" />}
+            <div>Older</div>
+            <div>logs</div>
+          </div>
+        </Button>
+      )}
+      <CustomScrollbar autoHide>
+        <div className={styles.pagesWrapper}>
+          <div className={styles.pagesContainer}>
+            {pages.map((page: LogsPage, index) => (
+              <div
+                className={styles.page}
+                key={page.queryRange.to}
+                onClick={() => !loading && changeTime({ from: page.queryRange.from, to: page.queryRange.to })}
+              >
+                <div className={classNames(styles.line, { selectedBg: currentPageIndex === index })} />
+                <div className={classNames(styles.time, { selectedText: currentPageIndex === index })}>
+                  {`${formatTime(oldestLogsFirst ? page.logsRange.from : page.logsRange.to)} - ${formatTime(
+                    oldestLogsFirst ? page.logsRange.to : page.logsRange.from
+                  )}`}
+                </div>
+              </div>
+            ))}
+          </div>
+          <div className={styles.filler}></div>
```

Comment:
you should be able to achieve the same without creating an element by doing

```css

      &::after {
        content: '';
        display: block;
        background: repeating-linear-gradient(
          135deg,
          ${theme.colors.bg1},
          ${theme.colors.bg1} 5px,
          ${theme.colors.bg2} 5px,
          ${theme.colors.bg2} 15px
        );
        width: 3px;
        height: inherit;
        margin-bottom: 8px;
      }
```

in `pagesWrapper` instead of creating a new element